src/pages/about/[service].tsx

```diff
@@ -0,0 +1,88 @@
+import { promises as fs } from 'fs'
+import path from 'path'
+import yaml from 'js-yaml'
+import { GetStaticProps } from 'next'
+import { parseMarkdown } from '@/utils/cms'
+import { LandingPage } from '@/components/LandingPage'
+import { LandingPageSchema } from '@/cms/landing-page.schema'
+import '@gouvfr/dsfr/dist/dsfr/dsfr.css'
+
+export default function ServiceLandingPage({
+  id,
+  data,
+}: {
+  id: string
+  data: LandingPageSchema
+}) {
+  return <LandingPage {...{ id, data }} />
+}
+
+export const getStaticPaths = async () => {
+  const enabledServices = await fs
+    .readdir(path.join(process.cwd(), 'content', 'landing-pages', 'fr'))
+    .then((files) =>
+      files
+        .filter((file) => file.endsWith('.yml'))
+        .map((file) => file.replace(/\.yml$/, ''))
+    )
+
+  return {
+    paths: enabledServices.map((service) => ({ params: { service } })),
+    fallback: false,
+  }
+}
+
+export const getStaticProps: GetStaticProps = async (context) => {
+  if (!context.params?.service) {
+    throw Error('No service provided')
+  }
+
+  let serviceFile
+  try {
+    serviceFile = await fs.readFile(
+      path.join(
+        process.cwd(),
+        'content',
+        'landing-pages',
+        'fr',
+        `${context.params.service}.yml`
+      ),
+      'utf8'
+    )
+  } catch (error) {
+    throw Error(`No data found for ${context.params.service}`)
+  }
+
+  let service: LandingPageSchema
+  try {
+    service = await parseMarkdown(yaml.load(serviceFile) as LandingPageSchema)
+  } catch (error) {
+    console.log(error)
+
+    throw Error(`Error parsing ${context.params.service}.json file`)
+  }
+
+  // little tricks to alternate backgrounds if none are set,
+  // making sure to also alternate between blocks that have set bgs
+  service.flexible_content.forEach((block, i) => {
```

Comment:
`flexible_content` could be undefined.

<img width="853" alt="Capture d’écran 2024-06-18 à 13 39 53" src="https://github.com/numerique-gouv/lasuite-landingpage/assets/45729124/462a8be7-8a2b-4900-bc97-7412f247ceca">

<img width="1512" alt="Capture d’écran 2024-06-18 à 13 38 37" src="https://github.com/numerique-gouv/lasuite-landingpage/assets/45729124/f610a97a-4136-48f6-abd5-915ccdf0ad37">

In the preview
<img width="1512" alt="Capture d’écran 2024-06-18 à 13 33 56" src="https://github.com/numerique-gouv/lasuite-landingpage/assets/45729124/89127350-9899-42db-affb-441541150524">

Reply:
Woops! Thanks